fog/ingest/enclave/impl/src/lib.rs

```diff
@@ -391,7 +392,7 @@ fn seal_private_key(src: &RistrettoPrivate) -> Result<SealedIngestKey> {
 // Helper for converting error type living only in mc_attest_trusted to the
 // error type in enclave_api (The attest_trusted crate will not compile in
 // non-sgx environment.)
-fn map_sealing_error(src: mc_attest_trusted::IntelSealingError) -> Error {
+fn map_sealing_error(src: IntelSealingError) -> Error {
```

Comment:
we might be able to redo this function, this was a workaround from the time when consensus and fog were in separate repos and this error was not in attest core yet

Reply:
like we could replace this with a `From<IntelSealingError> for Error` and then drop the `map_err(map_sealing_error)` part

Reply:
I think b4c1879 should address this